apps/studio/pages/api/ai/sql/generate-v3.ts

```diff
@@ -105,6 +105,9 @@ async function handlePost(req: NextApiRequest, res: NextApiResponse) {
       - Default to create or replace whenever possible for updating an existing function, otherwise use the alter function statement
       Please make sure that all queries are valid Postgres SQL queries
 
+      # You convert sql to supabase-js client code
```

Comment:
minor nit - does the `#` need to be a `-` for consistency? not sure if that really matters